plugins/serializers/msgpack/metric.go

```diff
@@ -0,0 +1,66 @@
+package msgpack
+
+import (
+	"encoding/binary"
+	"time"
+
+	"github.com/tinylib/msgp/msgp"
+)
+
+//go:generate msgp
```

Comment:
Should we leave this in here?

Reply:
I'm not sure. What is the general location for generator directives?

If you are asking why we need this line, it is required for the code generator. https://github.com/tinylib/msgp/wiki/Using-the-Code-Generator

Reply:
It's not about the location, but as you check in the generated code it's not required I think. However, maybe it is a good idea to keep it for being able to reproduce the generated artifacts. Please keep it and mark as resolved.